### DISCUSSION TITLE
Android端末で閲覧するとコードブロックの `1` と `l` の見分けが付きづらい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#592] [要望・アイデア・不具合]
Android 端末で Qiita を閲覧する際、コードブロック内の `1`（イチ）と `l`（エル）の見分けが非常に付きづらいです。

    1l1l1l1l1l1l1l1l1l1l1l1l1l1l1l1l

![image](https://github.com/increments/qiita-discussions/assets/75878492/dc3a9641-472f-4dc8-836d-22acefef1ef9)

参考：https://qiita.com/Yarakashi_Kikohshi/private/a1ab9101ac633b0882f3

実際に利用されているフォントがどれになっているのか分かりませんが、見分けやすいフォントになっていると良いなぁと思います。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます🙏
Android端末で確認したところ、確かにChromeだと見分けづらそうです…

我々としてもできるだけ見やすくしたいと思ってはいるのですが、
環境によって表示が異なり、全ての環境の表示を我々でコントロールすることは難しいため
大変申し訳ないのですが、別のブラウザをご利用いただく等で対応いただけると幸いです🙇

今回のご要望と直接の繋がりはないのですが、実装の難しさなどの詳細な背景は下記Discussions もご参照いただけると幸いです🙇
https://github.com/increments/qiita-discussions/discussions/302#discussioncomment-4814346

**Reply by yarakos95** [reply]
Android の Chromium 系ブラウザでしか確認していませんでしたが、（DL していたのを忘れていた）Firefox で閲覧すると見分けやすいフォントでした。これは確認不足でした 🙇‍♂️

今回の Discussion 提起で、`1` と `l` の見分けが付きづらいことが把握されたことだけでも満足です。
挙げられている Discussion で等幅フォントに非常に苦慮されていることは存じていますが、いつか打開策が見つけられた際には Android の Chromium 系ブラウザでも見やすいフォントであることを期待しています 😊
